runner.py

```diff
@@ -44,9 +44,18 @@ def get_configs(path: Path) -> List[str]:
                         default='configs/config_example.json',
                         help='The path to a configuration file or '
                              'a directory that contains configuration files')
+    parser.add_argument('--device', '--devices', default='host cpu gpu None', type=str, nargs='+',
+                        choices=('host', 'cpu', 'gpu', 'None'),
+                        help='Availible execution context devices. '
+                        'This parameter only marks devices as available, '
+                        'make sure to add the device to the config file '
+                        'to run it on a specific device')
     parser.add_argument('--dummy-run', default=False, action='store_true',
                         help='Run configuration parser and datasets generation '
                              'without benchmarks running')
+    parser.add_argument('--dtype', '--dtypes', type=str, default="float32 float64", nargs='+',
+                        choices=("float32", "float64"),
```

Comment:
Out of PR scope, but int32 might be added

Reply:
It is better to add this kind of changes in other PR